package.json

```diff
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "@hono/zod-openapi": "^0.17.0",
+    "@node-rs/argon2": "^2.0.0",
```

Comment:
This dependency is no more required and was removed in a previous MR, please take the latest pull from main.